x-pack/plugins/observability_solution/inventory/public/pages/inventory_page/index.tsx

```diff
@@ -62,47 +30,10 @@ export function InventoryPage() {
     return () => {
       searchBarContentSubscription.unsubscribe();
     };
-  });
-
-  function handlePageChange(nextPage: number) {
-    inventoryRoute.push('/', {
-      path: {},
-      query: { ...query, pageIndex: nextPage },
-    });
-  }
-
-  function handleSortChange(sorting: EuiDataGridSorting['columns'][0]) {
-    inventoryRoute.push('/', {
-      path: {},
-      query: {
-        ...query,
-        sortField: sorting.id as EntityColumnIds,
-        sortDirection: sorting.direction,
-      },
-    });
-  }
-
-  function handleTypeFilter(entityType: EntityType) {
-    inventoryRoute.push('/', {
-      path: {},
-      query: {
-        ...query,
-        // Override the current entity types
-        entityTypes: [entityType],
-      },
-    });
-  }
+    // If query has updated, the inventoryRoute state is also updated
+    // as well, so we only need to track changes on query.
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [query, searchBarContentSubject$]);
 
-  return (
-    <EntitiesGrid
-      entities={value.entities}
-      loading={loading}
-      sortDirection={sortDirection}
-      sortField={sortField}
-      onChangePage={handlePageChange}
-      onChangeSort={handleSortChange}
-      pageIndex={pageIndex}
-      onFilterByType={handleTypeFilter}
-    />
-  );
+  return query.view === 'unified' ? <UnifiedInventory /> : <GroupedInventory />;
```

Comment:
if the view is not specified, by default we'll show GroupedInventory. Is this the expected behaviour ? 

wdyt of passing the default value in the template ? https://github.com/elastic/kibana/pull/195475/files#diff-120bfc5b8cb6d2c04aa051f2955c29a701b20938653454fb353aedb2d2762e21R44